source/developers-guide/event-list/index.md

```diff
@@ -0,0 +1,2025 @@
+        ];
+    }
+
+###Listener
+    public function onEnlightControllerActionPostDispatch(\Enlight_Event_EventArgs $arguments)
+    {
+        /** @var $enlightController Enlight_Controller_Action */
+        $enlightController = $arguments->getSubject();
+     
+        /** @var $request Enlight_Controller_Request_RequestHttp */
+        $request = $arguments->getRequest();
+     
+        /** @var $response Enlight_Controller_Response_ResponseHttp */
+        $response = $arguments->getResponse();
+    }
+
+###Arguments dump
+    stdClass Object
+    (
+        [__CLASS__] => Enlight_Event_EventArgs
+        [_processed] => 
+        [_name] => Enlight_Controller_Action_PostDispatch
+        [_return] => 
+        [_elements] => Array
+            (
+                [subject] => stdClass Object
+                    (
+                        [__CLASS__] => Shopware_Proxies_ShopwareControllersFrontendIndexProxy
+                        [front] => Shopware_Proxies_EnlightControllerFrontProxy
+                        [view] => Enlight_View_Default
+                        [request] => Enlight_Controller_Request_RequestHttp
+                        [response] => Enlight_Controller_Response_ResponseHttp
+                        [controller_name] => Frontend_Index
+                        [instances] => Array(0)
+                    )
+     
+                [request] => stdClass Object
+                    (
+                        [__CLASS__] => Enlight_Controller_Request_RequestHttp
+                        [_paramSources] => Array(2)
+                        [_requestUri] => /sw406/
+                        [_baseUrl] => /sw406
+                        [_basePath] => /sw406
+                        [_pathInfo] => /
+                        [_params] => Array(0)
+                        [_rawBody] => 
+                        [_aliases] => Array(0)
+                        [_dispatched] => 1
+                        [_module] => frontend
+                        [_moduleKey] => module
+                        [_controller] => index
+                        [_controllerKey] => controller
+                        [_action] => index
+                        [_actionKey] => action
+                    )
+     
+                [response] => stdClass Object
+                    (
+                        [__CLASS__] => Enlight_Controller_Response_ResponseHttp
+                        [_cookies] => Array(0)
+                        [_body] => Array(0)
+                        [_exceptions] => Array(0)
+                        [_headers] => Array(1)
+                        [_headersRaw] => Array(0)
+                        [_httpResponseCode] => 200
+                        [_isRedirect] => 
+                        [_renderExceptions] => 
+                        [headersSentThrowsException] => 1
+                    )
+            )
+    )
+
+##Enlight_Controller_Action_PostDispatch_[Frontend/Backend]_*
+
```

Comment:
`__CLASS__ . '_PostDispatch_' . $moduleName,` is missing

all the PostDispatchSecure Events are missing